Sources/View/NibLoadable.swift

```diff
@@ -14,6 +14,7 @@ import UIKit
 ///  * they *are* NIB-based, and
 ///  * this class is used as the XIB's root view
 ///
+///  * this class is used also for instante UIViewController from XIB's as the XIB's File's Owner
```

Comment:
This bullet point doesn't make sense grammatically with the rest of the sentence 🤔 :
> Make your UIView subclasses conform to this protocol when:
>…
> * this class is used also for instant UIViewController from XIB's…

Instead maybe drop the bullet point and start a new sentence instead of continuing the previous list. Eg:

> You can also mark your UIViewController subclasses conform to this protocol to be able to instantiate them from the XIB with the same name as the class

Reply:
I had not seen this discussion either. I think this branch is really no longer needed right now.

Thank you anyway.